packages/@aws-cdk/cicd/README.md

```diff
@@ -0,0 +1,31 @@
+## Continuous Integration / Continuous Delivery for CDK Applications
+
+### Getting Started
+In order to add the `PipelineDeployStackAction` to your *CodePipeline*, you need to have a *CodePipeline* artifact that
+contains the result of invoking `cdk synth` on your *CDK App*. You can for example achieve this using a *CodeBuild*
```

Comment:
You should mention that the artifact must be a _directory_ that contains the output of `cdk synth -o <dir>`. Not just a single template.

Reply:
True